commons-text/src/main/java/org/jdbi/v3/commonstext/StringSubstitutorTemplateEngine.java

```diff
@@ -22,9 +22,29 @@
  * Register an instance of this class ({@link org.jdbi.v3.core.statement.SqlStatements#setTemplateEngine}) to use an Apache Commons Text {@link StringSubstitutor} as a {@link TemplateEngine}. This lets you use any pair of strings as variable delimiters, enabling the use of syntax like <pre>select * from ${foo}</pre>, <pre>select * from &lt;foo&gt;</pre>, <pre>select * from %foo%</pre>, etc.
  */
 public class StringSubstitutorTemplateEngine implements TemplateEngine {
+    /**
+     * @deprecated use the default constructor instead
+     */
+    @Deprecated
+    public static final TemplateEngine INSTANCE = new StringSubstitutorTemplateEngine();
+
```

Comment:
This is a new member in this PR, no?

I don't see it in the existing Javadoc: http://jdbi.org/apidocs/org/jdbi/v3/commonstext/StringSubstitutorTemplateEngine.html

Could just be removed.

Reply:
Ah, chicken that I am, I added it in the previous commit for the test when I was still holding onto factories and consts

Reply:
Thanks for catching that, would've been an addition to the regrets if that had gotten through 😁